sdks/js/src/types.ts

```diff
@@ -3019,3 +3019,16 @@ const MCP_VALIDATION_OUTPUTS = [
 ] as const;
 export type MCPValidationOutputPublicType =
   (typeof MCP_VALIDATION_OUTPUTS)[number];
+
+export const PostSpaceMembersRequestBodySchema = z.object({
+  userId: z.string(),
```

Comment:
We usualy use one single endpoint to add/remove multiple members at the same time ( like `PatchAgentEditorsRequestBodySchema`), with something like : 
```
  t.partial({
    addMemberIds: t.array(t.string),
    removeMemberIds: t.array(t.string),
  })
```
This is easier than calling multiple post, and have a distinct delete endpoint.